src/style.scss

```diff
@@ -40,47 +40,47 @@ $auro-inset-directions:'--squish';
 $auro-rgb-color-base-black-03: rgba(0, 0, 0, .03);
 $auro-rgb-color-base-white-03: rgba(255, 255, 255, .03);
 
-@import "./../node_modules/@aurodesignsystem/webcorestylesheets/dist/utilityMixins/insetUtility";
+@import "./../node_modules/@aurodesignsystem/webcorestylesheets/src/utilityMixins/insetUtility";
 
 // focus-visible mixin
 @mixin focus-visible {
   outline: 3px solid transparent;
-  box-shadow: inset 0 0 0 1px var(--auro-color-border-focus-on-dark), inset 0 0 0 3px var(--auro-color-background-lightest);
+  box-shadow: inset 0 0 0 1px var(--ds-color-base-white, $ds-color-base-white), inset 0 0 0 3px var(--ds-color-background-lightest, $ds-color-background-lightest);
 
   // auro-button--secondary
   &[variant='secondary'],
   &--secondary {
-    background-color: var(--auro-color-background-lighter);
-    box-shadow: inset 0 0 0 3px var(--auro-color-ui-default-on-light);
+    background-color: var(--ds-color-background-lighter, $ds-color-background-lighter);
+    box-shadow: inset 0 0 0 3px var(--ds-color-ui-default-default, $ds-color-ui-default-default);
   }
 
   // auro-button--tertiary
   &[variant='tertiary'],
   &--tertiary {
-    background-color: var(--auro-color-background-lighter);
-    border-color: var(--auro-color-ui-default-on-light);
-    box-shadow: inset 0 0 0 3px var(--auro-color-ui-default-on-light);
+    background-color: var(--ds-color-background-lighter, $ds-color-background-lighter);
+    border-color: var(--ds-color-ui-default-default, $ds-color-ui-default-default);
+    box-shadow: inset 0 0 0 3px var(--ds-color-ui-default-default, $ds-color-ui-default-default);
   }
```

Comment:
All focus-visible references can be removed. This is an old utility we don't support anymore.